mubench.cli/src/main/java/de/tu_darmstadt/stg/mubench/cli/ClassPath.java

```diff
@@ -0,0 +1,33 @@
+package de.tu_darmstadt.stg.mubench.cli;
+
+public class ClassPath {
+    private String path;
+
+    public ClassPath(String path) {
+        this.path = path;
+    }
+
+    public String[] getPaths() {
+        if (path != null && !path.isEmpty()) {
+            return path.split(":");
+        } else {
+            return new String[0];
+        }
+    }
+
+    public String getClasspath() {
+        return path;
+    }
+
+    public void append(ClassPath other){
```

Comment:
I would prefer `ClassPath` to be a value object (make `path` final) and `append()` to return a new `ClassPath` instance. Otherwise people may modify the inputs given to the detector, which may cause confusion. I see that `append` may not be an ideal name then, because it suggests modification (as for lists). Do you have a suggestion for a better name? (join, plus, ...)